### ISSUE TITLE
Bug: Payment Method Styles Not Specific Enough

### ISSUE THREAD

**Issue by jaswsinc** [woothemes/storefront#414]
Default Storefront theme with Stripe extension (screenshot below).

- Styles are applied to fieldsets and labels deeply which has a negative impact on fieldset and label tags used by the Stripe add-on. Notice the bullet icons, reduced font size, and even the background color for error notices is causing problems in this area of the checkout form.

![2016-06-10_02-11-30](https://cloud.githubusercontent.com/assets/1563559/15961356/cb85fb28-2eb0-11e6-9a23-cb16f932c6b3.png)

**Comment by jaswsinc** [comment]
Just sharing a short-term fix I added to a custom `.scss` file of my own.

```scss
.payment_method_stripe {
  > .stripe-legacy-payment-fields {
    padding:          0;

    > * fieldset {
      padding:        0;

      li {
        background:   none;
        font-size:    1.286em;
      }

      label {
        padding:      0;
        font-size:    1.286em;

        &::before {
          display:    none;
        }
      }
    }
  }
}
```

**Comment by jameskoster** [comment]
Fixed in #407 ;)

**Comment by jaswsinc** [comment]
Awesome! Thanks.
